fibonacci.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Template Name: fibonacci
+ * Template Post Type: post, page
+ *
+ * @package WordPress
+ * @subpackage fibonacci
+ * @since fibonacci
+ */
```

Comment:
Don't use unnecessary comments.
Use English only.
Use arguments and return type hints.